docs/docs-content/user-management/saml-sso/palette-sso-with-entra-id.md

```diff
@@ -0,0 +1,437 @@
+          - Browse to Properties, Edit Last Name
+          - Browse to Properties, add your corporate email address
+          - Add this account to the Entra ID Group `palette_tenant_admins`
+        
+18.	Change your Entra ID admin account with group membership. This is the account you used to create the **App** in Microsoft Entra ID. Add the following groups to this account, `palette_tenant_admins` and `k8s_cluster_admins`. 
+
+
+
+#### Configure Palette SSO 
+
+19. At this point should have at least four Entra ID user objects, including the admin account used to access first access Entra ID. Your Palette tenant will need information from Microsoft to complete the OIDC setup. Navigate back to the browser tab with Palette open.
+
+20. From the left **Main Menu** select **Users and Teams**. Next, choose **Teams** and then select **Create New Team**.
+
+21. Create four Palette teams with each team named after the **Entra ID Group ID*** you created in the previous steps.  Use the table below as an example reference.
+
+    | Palette Team Name is the Entra Group ID | Entra ID Group Name     |  
+    |-------------------|-------------------------------------------------|
+    | `e3ac07cc-bd12-4edc-92a4-983d783153ba` | `palette_tenant_admins`     |  
+    | `88f61c49-1dd1-40c3-a820-68a513a38725` |  `palette_default_project_admins` |   
+    | `3f33c3a5-e0af-4ef6-9671-c7545fe264f3` |  `k8s_cluster_admins`     |      
+    | `c4606295-e8b0-4df0-891b-de4428d7e54f` |  `k8s_cluster-editor`     |    
+
+22. Assign the following **Roles** to each Palette team. For example purposes, the same Entra Group ID values used in the table above are used in the table below.
+
+    | Palette Team Name | Role    | Entra ID Group Name |
+    |-------------------|---------| ------------------- |
+    | `e3ac07cc-bd12-4edc-92a4-983d783153ba` |  Tenant Admin     |   `palette_tenant_admins` |
+    | `88f61c49-1dd1-40c3-a820-68a513a38725` |  Project Admin   |   `palette_default_project_admins` |
+    | `3f33c3a5-e0af-4ef6-9671-c7545fe264f3` |  No role assigned    |   `k8s_cluster_admins` |
+    | `c4606295-e8b0-4df0-891b-de4428d7e54f` |  No role assigned   |  `k8s_cluster-editor` |
+
+Refer to the images below to ensure you have the correct settings.
+
+    ![Example of IDs](/oidc-entra-id-images/ba_tenantadmin.png)
+
+
+    ![Example of IDs with Palette roles attached to a team](/oidc-entra-id-images/25_def_project_admin.png)
+
+
+#### Palette SSO OIDC Configuration
+
+23. Navigate to left **Main Menu**, select **Tenant Settings**. Next, click on **SSO** and select the **OIDC** tab. 
+
+24. You will now configure the OIDC settings in Palette. Use the table below as a reference and populate the fields with the information you saved from the previous steps.
+
+
+| Field| Description |
+|------|-------------|
+| **Issuer URL**        | Add your tenant URL. The tenant URL looks like the following but with your unique tenant ID at the end `https://sts.windows.net/xxxxxxx`|
```

Comment:
Per Microsoft Entra's [documentation](https://learn.microsoft.com/en-us/entra/identity-platform/v2-protocols-oidc#find-your-apps-openid-configuration-document-uri), you are supposed to query the well-known Open ID configuration document API to determine the issuer url.
E.g.
```bash
curl -s https://login.microsoftonline.com/7fc4bd49-b7eb-410f-8ad4-11ab94561548/v2.0/.well-known/openid-configuration | jq '.issuer'
"https://login.microsoftonline.com/7fc4bd49-b7eb-410f-8ad4-11ab94561548/v2.0"
```